hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -120,7 +120,9 @@ public ContainerBalancer(
     this.containerManager = containerManager;
     this.replicationManager = replicationManager;
     this.ozoneConfiguration = ozoneConfiguration;
-    this.config = new ContainerBalancerConfiguration(ozoneConfiguration);
+    this.config = ozoneConfiguration.
+        getObject(ContainerBalancerConfiguration.class);
+    config.initialize(ozoneConfiguration);
```

Comment:
may be we can add a static build function to  `ContainerBalancerConfiguration` class to build and initialize a ContainerBalancerConfiguration instance, for example:
```
public static ContainerBalancerConfiguration buildFrom(ozoneConfiguration oc) {
.....
}
```